Spectrum/Windows/VJHUDWindow.xaml

```diff
@@ -379,6 +379,7 @@
                 <ComboBoxItem x:Name="domeActiveVisualizerRace" Content='Race' />
                 <ComboBoxItem x:Name="domeActiveVisualizerSnakes" Content='Snakes' />
                 <ComboBoxItem x:Name="domeActiveVisualizerQuaternionTest" Content='Quaternion Test' />
+                <ComboBoxItem x:Name="domeActiveVisualizerSplat" Content='Splat' />
```

Comment:
Maybe use the same name for consistency between `MainWindow` and `VJHudWindow`? Unless there's spacing issues or something